services/orderService.js

```diff
@@ -0,0 +1,16 @@
+const orderDao = require("../models/orderDao");
+
+const sendOrder = async (userId, totalPrice) => {
+  const { v4 } = require("uuid");
+
+  const uuid = () => {
+    const tokens = v4().split("-");
+    return tokens[2] + tokens[1] + tokens[0] + tokens[3];
+  };
+  const orderNumber = uuid();
```

Comment:
v4() 함수 결과를 orderNumber로 사용하면 될 것 같은데, split을 하는 이유가 있나요?
```suggestion
  const orderNumber = v4();
```